src/ripple/app/ledger/impl/LedgerConsensusImp.cpp

```diff
@@ -1175,10 +1176,15 @@ void LedgerConsensusImp::accept (std::shared_ptr<SHAMap> set)
                 newLCL, retriableTransactions, tapNONE);
         }
         for (auto const& item : localTx)
-            apply (accum, *item.second, tapNONE,
-                getApp().getHashRouter().sigVerify(),
+        {
+            if (getApp().getHashRouter().checkValidity(
+                *item.second).first != HashRouter::SigBad)
```

Comment:
We could be if the transaction isn't in the `HashRouter` or has expired, but we always were.

Before this change, we had to, because we had to keep the new open ledger away from other threads, and signature checking was in the Transactor. Now that it's not, we _could_ do the check before the lock is taken. However, I don't think it's worth it because the `localTx`s have already been checked, so the result is _likely_ to still be in `HashRouter`.